src/Microsoft.ML.Transforms/RandomFourierFeaturizing.cs

```diff
@@ -571,8 +565,8 @@ private ValueGetter<VBuffer<float>> GetterFromFloatType(IRow input, int iinfo)
                 var getSrc = input.GetGetter<float>(_srcCols[iinfo]);
                 var src = default(float);
 
-                var featuresAligned = new AlignedArray(RoundUp(1, _cfltAlign), CpuMathUtils.GetVectorAlignment());
-                var productAligned = new AlignedArray(RoundUp(_parent._transformInfos[iinfo].NewDim, _cfltAlign), CpuMathUtils.GetVectorAlignment());
+                var featuresAligned = new float[RoundUp(1)];
```

Comment:
Do you really need to `RoundUp` here anymore?

Reply:
yes this is required for using sse operations, where we will be loading things as a multiple of 4. I can rewriting these Factorizations algos in future to support any type of length but i dont think we should do it in this PR.

Reply:
yes we can change here to 4, sry I understand it as removing the ```RoundUp``` function entirely